php/php.editor/src/org/netbeans/modules/php/editor/indent/ui/Bundle.properties

```diff
@@ -430,3 +430,7 @@ FmtWrapping.groupUseListLabel.text=G&roup Use List:
 FmtWrapping.groupUseBracesCheckBox.text_1=Group &Use Braces
 FmtBraces.groupUseLabel.text=G&roup Use:
 FmtAlignment.noteLabel.text=<html><i>Note: These options depend on Wrapping settings</i></html>
+FmtBlankLines.maxPreservedField.text=
```

Comment:
Nitpick: The given text field can be set to be "NOI18N" in its properties (this line would not be here than).

Reply:
Fixed.